pyblish_lite/__init__.py

```diff
@@ -9,55 +6,8 @@
 version = '%i.%i.%i' % version_info
 __version__ = version
 
-
-def load_pyqt5():
-    import PyQt5.Qt
-    sys.modules["Qt"] = PyQt5
-    PyQt5.QtCore.Signal = PyQt5.QtCore.pyqtSignal
-    PyQt5.QtCore.Slot = PyQt5.QtCore.pyqtSlot
-    PyQt5.QtCore.Property = PyQt5.QtCore.pyqtProperty
-    print("Loaded PyQt5")
-
-
-def load_pyqt4():
-    import PyQt4.Qt
-    sys.modules["Qt"] = PyQt4
-    PyQt4.QtWidgets = PyQt4.QtGui
-    PyQt4.QtCore.Signal = PyQt4.QtCore.pyqtSignal
-    PyQt4.QtCore.Slot = PyQt4.QtCore.pyqtSlot
-    PyQt4.QtCore.Property = PyQt4.QtCore.pyqtProperty
-    print("Loaded PyQt4")
-
-
-def load_pyside2():
-    import PySide2
-    sys.modules["Qt"] = PySide2
-    print("Loaded PySide2")
-
-
-def load_pyside():
-    import PySide
-    from PySide import QtGui
-    PySide.QtWidgets = QtGui
-    sys.modules["Qt"] = PySide
-    print("Loaded PySide")
-
-
-# Support Qt 4 and 5, PyQt and PySide
-try:
```

Comment:
It was that way, but I ran into trouble on my end and reversed it into this.

I had trouble because my Maya had access to PyQt5, even though it shouldn't and can't use it. I think others will run into similar problem, and I'm not entire sure how to resolve it best.

Maybe by attempting to create a widget, and seeing if it throws an error?